GitVersionConfig.yaml

```diff
@@ -3,3 +3,5 @@ next-version: 6.0
 branches:
   develop:
     tag: alpha
+  releases?[/-]:
```

Comment:
Thanks @jakeginnivan, chimed in: https://github.com/GitTools/GitVersion/issues/1018#issuecomment-243085068